src/Symfony/Component/Security/Http/Logout/LogoutUrlGenerator.php

```diff
@@ -60,7 +60,7 @@ public function registerListener($key, $logoutPath, $csrfTokenId, $csrfParameter
      *
      * @return string The logout path
      */
-    public function getLogoutPath($key = null)
+    public function getLogoutPath(string $key = null)
```

Comment:
In these cases (e.g. accept some type or null) the type-hint should be added or not here and below?
(e.g. ?string)

Reply:
If `null` is the default, we omit the `?`.

Reply:
Thanks for the feedback! :) Is there anything else I should change?